NuKeeper.AzureDevOps/AzureDevOpsSettingsReader.cs

```diff
@@ -23,54 +52,27 @@ public RepositorySettings RepositorySettings(Uri repositoryUri)
                 return null;
             }
 
-            // general pattern is either of
-            // https://{org}.visualstudio.com/{project}/_git/{repo}/
-            // https://dev.azure.com/{org}/{project}/_git/{repo}/
-            // from this we extract owner and repo name
             var path = repositoryUri.AbsolutePath;
             var pathParts = path.Split('/')
                 .Where(s => !string.IsNullOrWhiteSpace(s))
                 .ToList();
 
-            if (pathParts.Count < 3)
+            if (pathParts.Count != 4)
```

Comment:
What kind of magic number is this? Maybe put in a property and have a descriptive name?

Reply:
Have move the comment on what this bit of code is trying to do closer, is this is descriptive enough now?

Reply:
Ah it makes sense when viewing the arrayparts below